pkgs/development/ocaml-modules/happy-eyeballs/mirage.nix

```
@@ -1,25 +1,39 @@
 { buildDunePackage
 , happy-eyeballs
+, duration
```

```suggestion
, duration
```
Where is duration used here?

---

`logs` not used too.
domain-name, fmt, ipaddr. maybe some more?